portfolio/src/main/webapp/index.html

```diff
@@ -34,13 +34,23 @@ <h1>Prashant's Portfolio</h1>
 
         <p>Go see some photos of me <a href="gallery.html">here</a>.</p>
 
-        <h3>Comment section:</h3>
+        <h5>Comment section:</h5>
 
         <div class="comment-section">
             <ul id="comments-container"></ul>
+            
+            <form onSubmit="displayNumComments();return false">
+                <label for="fname">How many comments would you like to see?</label><br>
+                <input type="number" step="1" id="comment-number"><br>
+                <input type="submit" />
+                <br>
+            </form><br/>
+
+            <button onClick="deleteAllComments()">Delete all comments</button><br/><br/><br/>
+
             <form action="/data" method="POST">
 
-                <p><strong>Contact me!</strong></p>
+                <h5>Contact me!</h5>
                 <p>What's your name?</p>
                 <textarea name="name-input" placeholder="Your name here"></textarea>
```

Comment:
the html <input> makes more sense here as someones name is not usually multiple lines long